### PR TITLE
Google sign in

### PR DESCRIPTION
**Pull request by ianshih2003**

### CODE REVIEW

File: src/screens/login/login.tsx [hunk 1]
```diff
@@ -37,24 +47,77 @@ const LoginScreen = ({
...
+
+      const token = await auth().currentUser?.getIdToken();
+
+      if (token) {
+        await saveToken(token);
+      }
+
+      await createNewUser(user);
+
+      navigation.push('Demo');
```
Comment (Ezequiel-Schejtman) [inline src/screens/login/login.tsx]:
```suggestion
      navigation.push('Workouts');
```

File: src/utils/constants.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import axios from 'axios';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
-export const API_URL = 'https://api-gateway-production.up.railway.app';
+export const API_URL = 'http://192.168.0.236:8080';
```
Comment (Ezequiel-Schejtman) [inline src/utils/constants.ts:4]: Can we leave this as is, with the deployed API URL?

File: src/screens/userProfile/userProfile.tsx [hunk 3]
```diff
@@ -120,6 +131,9 @@ const UserProfile = () => {
           onPress={() => setIsEditing(true)}>
           Edit
         </Button>
+        <Button mode='contained' style={styles.button} onPress={handleSignOut}>
+          Cerrar sesion
```
Comment (Ezequiel-Schejtman) [inline src/screens/userProfile/userProfile.tsx:135]: English. (See the login screen, we're using english all over the app).

**Review by Ezequiel-Schejtman — CHANGES_REQUESTED** [review]

File: src/utils/constants.ts (hunk 2 above)
Comment (ianshih2003) [reply src/utils/constants.ts:4]: Hay que hacer environments

Comment (ianshih2003) [reply src/utils/constants.ts:4]: https://dev.to/leon_arantes/react-native-multiple-environments-setup-schemaflavors-3l7p

Existian environments en react native

**Pull request merged**